tests/python/frontend/onnx/test_forward.py

```diff
@@ -30,6 +30,30 @@
 import scipy
 
 
+def get_tvm_output_with_vm(graph_def, input_data, target, ctx,
+                           opset=None, mode=None):
```

Comment:
Remove mode and always use vm

Reply:
Thanks for the suggestions and I updated it.